nb2workflow/galaxy.py

```diff
@@ -0,0 +1,662 @@
+
+import xml.etree.ElementTree as ET
+import os
+import re
+from textwrap import dedent
+import argparse
+import logging
+
+import yaml
+import json
+
+from oda_api.ontology_helper import Ontology
+from nb2workflow.nbadapter import NotebookAdapter, find_notebooks
+
+import nbformat
+from nbconvert.exporters import ScriptExporter
+
+from ensureconda.api import ensureconda
+import subprocess as sp
+
+import bibtexparser as bib
+import pypandoc
+
+import tempfile
+import black
+import autoflake
+import isort
+
+logger = logging.getLogger()
+        
+               
+default_ontology_path = 'http://odahub.io/ontology/ontology.ttl'
+
+global_req = []
+
+_success_text = '*** Job finished successfully ***'
+_ontology_base = 'http://odahub.io/ontology#'
```

Comment:
you only use it once in the next line, right?

Reply:
Actually yes. Probably it will be needed somewhere else, but I'm not sure, other ontology operations are through ontology_helper